src/renderers/common/nodes/Nodes.js

```diff
@@ -689,7 +696,11 @@ class Nodes extends DataMap {
 		const renderer = this.renderer;
 		const cacheKey = this.getOutputCacheKey();
 
-		const output = texture( outputTarget, screenUV ).renderOutput( renderer.toneMapping, renderer.currentColorSpace );
+		//builtin( 'gl_ViewID_OVR' )??
```

Comment:
@Mugen87 how can I incorporate the view id?

Reply:
Can you explain why you need to evaluate `gl_ViewID_OVR` at this place? I though it is just required to detect the correct camera matrices for the current view?

Reply:
Because we rendered to a multiview rendertarget in the first step and are rendering to the multiview xr target, we need to select the right texture in the texture array.
Without using `gl_ViewID_OVR`, the fragment shader will always pick the texture for the left eye.

Reply:
@Mugen87 I fixed this in my latest diff. It was easier than I thought :-)
This node system is quite the change! Very nice